From c195d3a9036d41dba42a86b61d9de342de074475 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Edgar=20Andr=C3=A9s=20Margffoy=20Tuay?= <andfoy@gmail.com>
Date: Thu, 17 Feb 2022 15:17:23 -0500
Subject: [PATCH] Make v8 binaries compatible with x86_64 when cross-compiling
 on arm64

---
 chromium/v8/gni/v8.gni | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/chromium/v8/gni/v8.gni b/chromium/v8/gni/v8.gni
index 13ffc2c3391..88065ea85bf 100644
--- a/chromium/v8/gni/v8.gni
+++ b/chromium/v8/gni/v8.gni
@@ -226,6 +226,10 @@ template("v8_executable") {
       # For enabling ASLR.
       ldflags = [ "-pie" ]
     }
+    
+    ldflags += ["-arch x86_64 -arch arm64"]
+    cflags += ["-arch x86_64 -arch arm64"]
+    
     if (defined(testonly) && testonly && v8_code_coverage) {
       # Only add code coverage cflags for non-test files for performance
       # reasons.
-- 
2.25.1

