From 6a1d81a09ff0c05cbdf649db2a4c88a5070d41e4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Edgar=20Andr=C3=A9s=20Margffoy=20Tuay?= <andfoy@gmail.com>
Date: Wed, 23 Feb 2022 16:04:03 -0500
Subject: [PATCH] Compile protoc for both x86_64 and arm64

---
 chromium/third_party/protobuf/BUILD.gn | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/chromium/third_party/protobuf/BUILD.gn b/chromium/third_party/protobuf/BUILD.gn
index 4ebf4573382..c4e22ac0370 100644
--- a/chromium/third_party/protobuf/BUILD.gn
+++ b/chromium/third_party/protobuf/BUILD.gn
@@ -384,6 +384,8 @@ static_library("protobuf_full") {
   ]
 
   cflags = protobuf_lite_cflags
+  cflags += ["-arch", "arm64", "-arch", "x86_64"]
+  ldflags = ["-arch", "arm64", "-arch", "x86_64"]
 
   defines = [ "HAVE_ZLIB" ]
 }
@@ -581,6 +583,8 @@ if (current_toolchain == host_toolchain) {
     cflags = protobuf_lite_cflags
 
     public_deps = [ ":protobuf_full" ]
+    cflags += ["-arch", "arm64", "-arch", "x86_64"]
+    ldflags = ["-arch", "arm64", "-arch", "x86_64"]
   }
 
   executable("protoc") {
@@ -597,6 +601,9 @@ if (current_toolchain == host_toolchain) {
       # Default manifest on Windows (a no-op elsewhere).
       "//build/win:default_exe_manifest",
     ]
+    
+    cflags += ["-arch", "arm64", "-arch", "x86_64"]
+    ldflags = ["-arch", "arm64", "-arch", "x86_64"]
   }
 }
 
-- 
2.25.1

