diff --git a/src/3rdparty/chromium/v8/src/base/platform/platform-posix.cc b/src/3rdparty/chromium/v8/src/base/platform/platform-posix.cc
index 280d7f88f..d437e5730 100644
--- a/src/3rdparty/chromium/v8/src/base/platform/platform-posix.cc
+++ b/src/3rdparty/chromium/v8/src/base/platform/platform-posix.cc
@@ -88,6 +88,10 @@ extern int madvise(caddr_t, size_t, int);
 extern "C" void* __libc_stack_end;
 #endif
 
+#ifndef MFD_CLOEXEC
+# define MFD_CLOEXEC 0x0001U
+#endif
+
 namespace v8 {
 namespace base {
 
