diff --git a/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_address_space.cc b/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_address_space.cc
index 0040a037f34..46158a7efc9 100644
--- a/src/3rdparty/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_address_space.cc
+++ b/src/3rdparty/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_address_space.cc
@@ -361,7 +361,7 @@ base::PlatformFile CreateAnonymousFileForMapping(
     [[maybe_unused]] const char* name,
     [[maybe_unused]] size_t size) {
   base::PlatformFile fd = base::kInvalidPlatformFile;
-#if PA_BUILDFLAG(IS_LINUX) || PA_BUILDFLAG(IS_CHROMEOS)
+#if 0 //PA_BUILDFLAG(IS_LINUX) || PA_BUILDFLAG(IS_CHROMEOS)
   // TODO(crbug.com/40238514): if memfd_secret() is available, try
   // memfd_secret() first.
   fd = memfd_create(name, MFD_CLOEXEC);
diff --git a/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_root.cc b/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_root.cc
index a025fd2dc43..8f520248c5b 100644
--- a/src/3rdparty/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_root.cc
+++ b/src/3rdparty/chromium/base/allocator/partition_allocator/src/partition_alloc/partition_root.cc
@@ -1967,7 +1967,7 @@ PA_NOINLINE void PartitionRoot::QuarantineForBrp(
 // static
 #if PA_CONFIG(ENABLE_SHADOW_METADATA)
 void PartitionRoot::EnableShadowMetadata(internal::PoolHandleMask mask) {
-#if PA_BUILDFLAG(IS_LINUX)
+#if 0 //PA_BUILDFLAG(IS_LINUX)
   // TODO(crbug.com/40238514): implement ModuleCache() or something to
   // load required shared libraries in advance.
   // Since memfd_create() causes dlsym(), it is not possible to invoke
diff --git a/chromium/mojo/core/channel_linux.cc b/chromium/mojo/core/channel_linux.cc
index dd5edfce80e..364d263a26b 100644
--- a/src/3rdparty/chromium/mojo/core/channel_linux.cc
+++ b/src/3rdparty/chromium/mojo/core/channel_linux.cc
@@ -18,6 +18,27 @@
 #include <sys/utsname.h>
 #include <unistd.h>
 
+#ifndef F_SEAL_SEAL
+#define F_SEAL_SEAL   0x0001
+#endif
+#ifndef F_SEAL_SHRINK
+#define F_SEAL_SHRINK 0x0002
+#endif
+#ifndef F_SEAL_GROW
+#define F_SEAL_GROW   0x0004
+#endif
+#ifndef F_SEAL_WRITE
+#define F_SEAL_WRITE  0x0008
+#endif
+
+#ifndef F_ADD_SEALS
+#define F_ADD_SEALS 1033
+#endif
+
+#ifndef F_GET_SEALS
+#define F_GET_SEALS 1034
+#endif
+
 #include <algorithm>
 #include <atomic>
 #include <cstring>
